backend/src/schema/resolvers/rooms.ts

```diff
@@ -0,0 +1,56 @@
+import { v4 as uuid } from 'uuid'
+import { neo4jgraphql } from 'neo4j-graphql-js'
+import Resolver from './helpers/Resolver'
+
+export default {
+  Query: {
+    Room:  async (object, params, context, resolveInfo) => {
+      if (!params.filter) params.filter = {}
+      params.filter.users_some = {
+        id: context.user.id,
+      }
+      return neo4jgraphql(object, params, context, resolveInfo)
+    },
+  },
+  Mutation: {
+    CreateRoom: async (_parent, params, context, _resolveInfo) => {
+      const { userId } = params
+      const { user: { id: currentUserId } } = context
+      const roomId = uuid()
+      const session = context.driver.session()
+      const writeTxResultPromise = session.writeTransaction(async (transaction) => {
+        const createRoomCypher = `
+          MATCH (currentUser:User { id: $currentUserId })
+          MATCH (user:User { id: $userId })
+          MERGE (currentUser)-[:CHATS_IN]->(room:Room)<-[:CHATS_IN]-(user)
+          SET room.createdAt = toString(datetime()),
+          room.id = $roomId
+          RETURN room { .* }
+        `
+        const createRommTxResponse = await await transaction.run(
```

Comment:
How does this compile?
```suggestion
        const createRommTxResponse = await transaction.run(
```